io/private/las/Vlr.hpp

```diff
@@ -85,10 +85,10 @@ struct Vlr
 
     Vlr() = default;
     Vlr(const std::string& userId, uint16_t recordId, const std::string& description) :
-        userId(userId), recordId(recordId), description(description)
+        userId(userId), recordId(recordId), description(description), writeAsEVLR(false)
```

Comment:
This attribute belongs in EVLR, I think. It looks like it's only ever used in the context of an Evlr.

Reply:
I put it on the VLR because I was worried someone might be using the VLR class by itself. I'm not happy with the arrangement anyway, as it mixes up VLR and EVLR, but I wasn't quite sure what to do here 🤷.

Reply:
I think the logic, for better or worse, was that you would create an Evlr object in the system and it would get sliced to a Vlr object when you knew you wanted to write to the Vlr section rather than the Evlr section.

Reply:
yes, that's probably the behavior you want most of the time too. My situation is I want to attach large metadata documents to the end of files and there's no need for these to be up front (think COPC content).